test/run_pass/test_stream_pipe.js

```diff
@@ -0,0 +1,109 @@
+/* Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var assert = require('assert');
+var stream = require('stream');
+var Duplex = stream.Duplex;
+var Readable = stream.Readable;
+var Writable = stream.Writable;
+
+var readable = new Readable();
+
+var msg = [];
+var resp = [];
+
+// 1
```

Comment:
It's better to use a more informative comment. E.g. `// Step 1` or `// Test 1` or `// Testcase 1`